logstash-input-webfetcher/src/main/resources/logback.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
+<configuration>
+	<property name="USER_HOME"
+		value="/home/ouaalja/devel/java/code/logstash-inputs/logstash-input-webfetcher/" />
+
+
+	<appender name="FILE-THREAD"
+		class="ch.qos.logback.classic.sift.SiftingAppender">
+
+		<!-- This is MDC value -->
+		<!-- We will assign a value to 'logFileName' via Java code -->
+		<discriminator>
+			<key>logFileName</key>
+			<defaultValue>logstash-input-webfetcher</defaultValue>
+		</discriminator>
+
+		<sift>
+
+			<!-- A standard RollingFileAppender, the log file is based on 'logFileName' 
+				at runtime -->
+			<appender name="FILE-${logFileName}"
+				class="ch.qos.logback.core.rolling.RollingFileAppender">
+				<file>${USER_HOME}/${logFileName}.log</file>
+
+				<encoder
+					class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+					<Pattern>
+						%d{yyyy-MM-dd HH:mm:ss} %mdc [%thread] %level %logger{35}
+						- %msg%n
+					</Pattern>
+				</encoder>
+
+				<rollingPolicy
+					class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
+					<FileNamePattern>${USER_HOME}/${logFileName}.%i.log.zip
+					</FileNamePattern>
+					<MinIndex>1</MinIndex>
+					<MaxIndex>10</MaxIndex>
+				</rollingPolicy>
+
+				<triggeringPolicy
+					class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
+					<MaxFileSize>10MB</MaxFileSize>
+				</triggeringPolicy>
+
+			</appender>
+
+		</sift>
+	</appender>
+
+	<appender name="STDOUT"
+		class="ch.qos.logback.core.ConsoleAppender">
+		<layout class="ch.qos.logback.classic.PatternLayout">
+			<Pattern>
+				%-5level %logger{36} - %msg%n
+			</Pattern>
+		</layout>
+	</appender>
+
+	<if condition='isDefined("elk.destination")'>
+		<then>
+
+			<appender name="LOGSTASH"
+				class="net.logstash.logback.appender.LogstashTcpSocketAppender">
```

Comment:
do not send to LogstashTcpSocketAppender. recomended action is to leave on drive, and pick it up with filebeat